### ISSUE TITLE
Mail rejected due to new rule in exim_stage1 configuration

### ISSUE THREAD

**Issue by reneklootwijk** [MailCleaner/MailCleaner#113]
The latest update of the exim_stage1.conf_template contains a rule that adds a header which fails regularly resulting in rejecting mail. The line causing the issue is:
`add_header     = :at_start:${authresults {$primary_hostname}}`

The error as stated in the rejectlog is:
`temporarily rejected after DATA: failed to expand ACL string ":at_start:${authresults {$primary_hostname}}": "${authresults" is not a known operator (or a } is missing in a variable reference)`

**Comment by p-rolle** [comment]
Hello,
to fix a security issue affecting exim, we upgraded this package to the latest version. This required to update the package and modify a variable we used since its name changed. So the upgrade is done in 2 steps :
getting the new configuration file (which is hosted on Github)
upgrade the package
Your server only did the first step this can have several causes but should not have happened. (I only saw that on a client server who broke the apt configuration and the server was not able to update the package list).
To resolve this run :
apt-get update
apt-get --force-yes -y -o Dpkg::Options::="--force-confdef" -o Dpkg::Options::="--force-confold" -o Dpkg::Options::="--force-confmiss" install mc-exim 
/usr/mailcleaner/etc/init.d/mailcleaner restart
